src/handlers/localstorage/uniswap.ts

```diff
@@ -10,18 +18,31 @@ const ASSETS = 'uniswapassets';
 const LIQUIDITY = 'uniswapliquidity';
 const UNISWAP_POSITIONS = 'uniswapPositions';
 const UNISWAP_FAVORITES = 'uniswapFavorites';
+const UNISWAP_FAVORITES_METADATA = 'uniswapFavoritesMetadata';
 const uniswapLiquidityVersion = '0.2.0';
 const uniswapPositionsVersion = '0.1.0';
 
 export const uniswapAccountLocalKeys = [ASSETS, LIQUIDITY, UNISWAP_POSITIONS];
 
-export const getUniswapFavorites = (network: any) =>
+export const getUniswapFavorites = (network: any): Promise<EthereumAddress[]> =>
```

Comment:
can use `Network` type from `import { Network } from '@rainbow-me/helpers/networkTypes';` instead of `any`

Reply:
can we use `Network` for the rest of the times `network` variable is used in this file?